### DISCUSSION TITLE
Invoking jinja from the command line?

### DISCUSSION THREAD

**Discussion by dtheodor** [pallets/jinja#1749] [Ideas]
I often come across the need to fill in a template from a bash script, e.g. by doing something like `fill_template --template template_file --variables vars.yml`. I have not come across a lightweight script that can do this. It would be great if I could do this with jinja, with a basic CLI invocation being something like  `python -m jinja2  --template template_file --variables vars.yml`

**Comment by gone** [comment]
I use cookiecutter for this
